src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const discOverThreeDays = 20;
+  const discOverSevenDays = 50;
+
+  if (days >= 7) {
+    return days * rentCost - discOverSevenDays;
+  } else if (days >= 3) {
+    return days * rentCost - discOverThreeDays;
+  } else {
+    return days * rentCost;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean

Reply:
avoid such nesting `else if` conditions
This code is hard to read and maintain

Reply:
pass days into variables too